### DISCUSSION TITLE
Prioritizing topics for TRAPI 1.3 Phase 2 by vote

### DISCUSSION THREAD

**Discussion by edeutsch** [NCATSTranslator/ReasonerAPI#317] [General]
This discussion thread will serve as the voting platform for prioritizing the topics to tackle for phase 2 of TRAPI 1.3 development (next 4 months). After everyone votes, we will address them roughly in order of acquired votes.

All regular team representatives/attendees of the TRAPI call are strongly requested to vote. All other Translator participants are welcome to vote.

You are NOT voting for a particular solution.
You ARE voting to prioritize the topics for future TRAPI calls to be resolved and included for the TRAPI 1.3 release.

Voting Scheme:
+2 Highest (heart) - Must get this done for TRAPI 1.3
+1 High (thumb up) - Should try to get this done for TRAPI 1.3
0 Medium (none) - Meh, if there's extra time by the deadline, sure
-1 Low Priority (thumb down) - This should not be prioritized. Can take this up for TRAPI 1.4.

Results summary as of 2022-03-09 11pm PST:
 7 DONE. Adjust attributes to allow unlimited recursion (#315)
 6 AGREED. Execute a plan to release TRAPI 1.3 in mid-summer, after the May/June Relay
 4 DONE. Add NodeBinding.attributes (#312)
 4 What should the /meta_knowledge_graph for ARAs be? (policy clarification) (#288)
 4 Consider schema changes to facilitate merging
 2 Add QEdge.exclude flag (#306)
 1 Remodel EPC information separately from Attributes in conjunction with data modeling group
 0 Develop a detailed implementation plan for QEdge/Edge.negated (#307)

**Comment by edeutsch** [comment]
Add QEdge.exclude flag (#306) to allow the querier to exclude (discard) results that can have a certain relationship
e.g.: Find drugs related_to Disease X, excluding those that are contraindicated_for Disease X
e.g.: Find genes related to Disease X, excluding those that are related to Disease Y

Vote for this topic with an emoji: heart, thumbs up, nothing, thumbs down

**Comment by edeutsch** [comment]
Add NodeBinding.attributes (#312) 
Analagous to existing EdgeBinding.attributes

Vote for this topic with an emoji: heart, thumbs up, nothing, thumbs down

**Comment by edeutsch** [comment]
Develop a detailed implementation plan for QEdge/Edge.negated
We have a PR: https://github.com/NCATSTranslator/ReasonerAPI/pull/307
But it was deemed unwise to merge until there is a detailed implementation plan

Vote for this topic with an emoji: heart, thumbs up, nothing, thumbs down

**Comment by edeutsch** [comment]
Remodel EPC information separately from Attributes in conjunction with data modeling group

Vote for this topic with an emoji: heart, thumbs up, nothing, thumbs down

**Comment by edeutsch** [comment]
Adjust attributes to allow unlimited recursion (#315)

Vote for this topic with an emoji: heart, thumbs up, nothing, thumbs down

**Reply by brettasmi** [reply]
I strongly disagree with allowing _infinite_ recursion, but I support this otherwise :)

**Reply by edeutsch** [reply]
> I strongly disagree with allowing _infinite_ recursion, but I support this otherwise :)
okay, fair enough, updated to _unlimited_ recursion.

**Comment by edeutsch** [comment]
Consider schema changes to facilitate merging
See existing issue discussing the matter: #313

Vote for this topic with an emoji: heart, thumbs up, nothing, thumbs down

**Comment by edeutsch** [comment]
Execute a plan to release TRAPI 1.3 in mid-summer, after the May/June Relay

Vote for this topic with an emoji: heart, thumbs up, nothing, thumbs down

**Comment by edeutsch** [comment]
What should the /meta_knowledge_graph for ARAs be? (policy clarification)
https://github.com/NCATSTranslator/ReasonerAPI/issues/288

Vote for this topic with an emoji: heart, thumbs up, nothing, thumbs down
